.github/workflows/lint-pr.yml

```diff
@@ -0,0 +1,17 @@
+name: Lint PR
```

Comment:
why not put it on CI.yml ?

Reply:
To make it work only on PR, but if necessary, I can move it to CI.

Reply:
Yes please lets move it in CI